CHANGELOG.md

```diff
@@ -1,4 +1,9 @@
 #### 1.9.10-SNAPSHOT
+- **[CHANGE]** `Action.parallelTo` changed to `Action.along`.
+- **[CHANGE]** `Action.along` (formerly `Action.parallelTo`) and `Action.then` no longer unwrap the second action.
+- **[CHANGE]** `ParallelAction.along` (formerly `ParallelAction.parallelTo`) and `SequenceAction.then` simply add the second action to the group without unwrapping.
+- **[UPDATE]** Added `/` operator to Action, which performs the non-mutating version of along, wrapping the caller and argument in a new ParallelAction.
```

Comment:
`UPDATE` tag is reserved for dependencies' updates. New features such as this operator are marked with `FEATURE`.